bindings/go/cmd/main.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"database/sql"
+	"fmt"
+	_ "limbo"
+	"os"
+)
+
+func main() {
+	testLib := ":memory:"
+	conn, err := sql.Open("limbo", testLib)
```

Comment:
I do wonder if this should be `sqlite3`. This is what we did in the JDBC driver, although that was about making JDBC URLs compatible between SQLite and Limbo. Not sure that logic applies to Go? But basically, we'd like things to be a drop-in replacement where you switch the package from SQLite to Limbo and everything else just works.